src/components/TaskStatusIcon/index.tsx

```diff
@@ -10,39 +10,47 @@ const failurePurple = "#36367F";
 interface TaskStatusIconProps {
   status: string;
   size?: Size | number;
+  rightMargin?: boolean;
 }
 
 export const TaskStatusIcon: React.FC<TaskStatusIconProps> = ({
   status,
   size = Size.Default,
+  rightMargin,
```

Comment:
Unfortunately wrapping <Icon /> with styled wouldn't apply the generated className to the component even though wrapping Icon does.

Reply:
Interesting, would this not be a matter of capturing the `className` prop that's passed to `TaskStatusIcon` and then including it in the props applied to the returned `<Icon />`

Reply:
Great idea 😅